packages/gatsby-tinacms-mdx/src/to-mdx.ts

```diff
@@ -19,11 +19,8 @@ limitations under the License.
 import { MdxNode } from './mdx-node'
 import * as yaml from 'js-yaml'
 
-export function toMarkdownString(mdx: MdxNode) {
+export function toMdxString(mdx: MdxNode) {
   return (
-    '---\n' +
-    yaml.dump(mdx.rawFrontmatter) +
-    '---\n' +
-    (mdx.rawMarkdownBody || '')
+    '---\n' + yaml.dump(mdx.rawFrontmatter) + '---\n' + (mdx.rawMdxBody || '')
```

Comment:
Template literal here might be easier to read

Reply:
Yeah, I think you're right, this is a bit hard to read 😂

I vote for template literal

Reply:
Template literals didn't work out because of indentation. Prettier wanted this to be a one liner so I changed it to an array so the formatting is made slightly more obvious.